SonarQube.Bootstrapper/ArgumentProcessor.cs

```diff
@@ -200,14 +242,16 @@ private static IBootstrapperSettings CreatePostProcessorSettings(IList<string> b
         /// command line arguments
         /// </summary>
         /// <remarks>We don't want to forward these arguments to the pre- or post- processor</remarks>
-        private static List<string> RemoveBootstrapperArgs(string[] commandLineArgs)
+        private static IList<string> RemoveBootstrapperArgs(string[] commandLineArgs)
         {
             if (commandLineArgs == null)
             {
                 throw new ArgumentNullException("param");
             }
 
-            return commandLineArgs.Except(new string[] { BeginVerb, EndVerb }).ToList();
+            var excludedVerbs = new string[] { BeginVerb, EndVerb, InstallTargetsVerb };
+
+            return commandLineArgs.Where(arg => !excludedVerbs.Any(e => arg.StartsWith(e, StringComparison.Ordinal))).ToList();
```

Comment:
There was a bug here - a command line argument starts with a verb, it isn't a verb